docker/qgisserver/tests/functional/accesscontrol_allow_to_edit_test.py

```diff
@@ -19,8 +19,8 @@
 from shapely.geometry import LineString, box, shape
 
 from geomapfish_qgisserver.accesscontrol import Access, GeoMapFishAccessControl, OGCServerAccessControl
-from .accesscontrol_test import add_node_in_qgis_project, set_request_parameters
-
+from .accesscontrol_test import add_node_in_qgis_project, set_request_parameters, \
+    scoped_session, server_iface, qgs_access_control_filter
```

Comment:
Is that right?
They are never used as python way, just in fixture...

Reply:
Fixtures and their visibility are a bit odd in pytest. They don't require importing, but if you defined them in a test_*.py file, they'll only be available in that file.

Reply:
Then it's required?

Reply:
It might sounds good to move them one level up in conftest.py